### ISSUE TITLE
Setup and Use Troubleshooting Help

### ISSUE THREAD

**Issue by reidellawfirm** [x13a/Duress#31]
I have followed the instructions here an in other issues but I am still not getting any results when I put in an incorrect password 2 digits greater than my current password. I am able to use my fingerprint reader to open immediately after. I am not sure if there are additional accessibility settings or Google services I should change or revise but would appreciate any and all help.

I am running OneUI5 on Fold4. I previously had this working on my OnePlus 7pro on LineageOS.

I attached screenshots for any further tips or tricks. Please point me in the right direction.

![Screenshot_20221231_183829_Duress](https://user-images.githubusercontent.com/75231847/210158347-6f90a126-d506-4bdb-8f4a-30ed210b136c.jpg)
![Screenshot_20221231_183932_Wasted](https://user-images.githubusercontent.com/75231847/210158351-5fd92093-25a7-4732-b595-985cc5ad8b9a.jpg)

**Comment by reidellawfirm** [comment]
I am willing to compensate anyone for their time in troubleshooting this with me. Just let me know, thanks!

**Comment by achemdcoduer** [comment]
have anoyone help you?

**Comment by reidellawfirm** [comment]
No, app never worked for me since moving to Samsung OneUI. I just deleted app after while. It seems abandoned.
